diamm/serializers/search/source.py

```diff
@@ -140,3 +141,10 @@ def get_public_images_b(self, obj):
             return True
         else:
             return False
+
+    def get_geographic_area_ii(self, obj):
+        geographic_area = list()
```

Comment:
obj.provenance.values_list(...) will give me a separate tuple for each dictionary though and it is an error to use the `flat` parameter when there is more than one field.

The use case for this is as you mentioned in issue #59:

> 1. Create a multi-valued array of all the `GeographicArea` records that are present on a source when indexing that source. When a city record is retrieved (https://beta.diamm.ac.uk/cities/246/) we can then do a Solr query for, e.g., `type:source geographic_area_ii:246` to retrieve all the sources that are related to that area.